ansible/roles/real-postfix/templates/postfix/virtual-alias-maps-relay-user-ldap.cf.j2

```diff
@@ -0,0 +1,10 @@
+# Check if the user's vhost is served from another pool; if so, we
+# relay to that pool's IP.
+
+server_host = {{ ldap_server }}
+search_base = ou=VirtualHosts,dc=scripts,dc=mit,dc=edu
+query_filter = (&(objectClass=scriptsVhost)(scriptsVhostName=%u.scripts.mit.edu)(!(|{% for ip in ansible_all_ipv4_addresses %}(scriptsVhostPoolIPv4={{ip}}){% endfor %})))
```

Comment:
Long-ish line (at least as rendered; the template is relatively sane)

Reply:
Also can't fix :(